src/components/Viewer/Viewer.js

```diff
@@ -13,13 +13,31 @@ import OpenSeadragon, { Point } from "openseadragon";
 /** @jsx jsx */
 import { jsx, css } from "@emotion/core";
 
-const topBarWrapper = css`
+const osdTopRow = css`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: rgba(0, 0, 0, 0.8);
+  position: absolute;
+  z-index: 100;
+  width: 100%;
+  height: 60px;
+`;
+const workTitle = css`
+  text-align: left;
+  color: white;
+  font-size: 1.25rem;
+  padding-left: 1rem;
+`;
+const toolBarWrapper = css`
   font-size: 1rem;
-  background: rgba(0, 0, 0, 0.5);
+  /* background: rgba(0, 0, 0, 0.5); */
```

Comment:
Can we remove this? Another minor change, in README.md for the project, one of the yarn dependency installs is missing `add`.

Reply:
Ah, good catch.  Yea, updating now.